tools/skyline2isocor/skyline2isocor.xml

```diff
@@ -0,0 +1,39 @@
+<tool id='skyline2isocor' name='Skyline2Isocor: converting skyling outputs into IsoCor inputs' version='@TOOL_VERSION@+galaxy0' profile='23.0'>
+    <macros>
+        <token name='@TOOL_VERSION@'>0.1.2</token>
+    </macros>
+    <requirements>
+        <requirement type='package' version='@TOOL_VERSION@'>skyline2isocor</requirement>
+    </requirements>
+    <command detect_errors='exit_code'><![CDATA[
+    skyline2isocor '$skyline_output' '$isocor_input' > '$stdout' 2> '$stderr'
+    ]]></command>
+    <inputs>
+        <param type='data' format='tabular, txt' name='skyline_output' label='Skyline output file' help='Skyline output to convert'/>
+    </inputs>
+    <outputs>
+        <data format='txt' name='stdout' label='Log'/>
```

Comment:
I would not expose stderr and stdout as separate files, Galaxy captures those differently.

But I guess this is a community decision.

Reply:
Agree with this point.

Otherwise, It seems in a good shape for me.

Reply:
Ok, should I thus change the wrapper so that you can merge the PR?